### DISCUSSION TITLE
Ollama integration?

### DISCUSSION THREAD

**Discussion by mars-s** [TheR1D/shell_gpt#362] [Ideas]
surely we can get a local method of running this with https://github.com/jmorganca/ollama

**Comment by TheR1D** [comment]
It is possible to run other LLMs locally using [LocalAI](https://github.com/mudler/LocalAI) or other OpenAI compatible local backends. In the config file we can point `sgpt` requests to **localhost** by changing `OPENAI_BASE_URL`.
